core/chains/evm/txmgr/eth_confirmer_test.go

```diff
@@ -745,6 +745,73 @@ func TestEthConfirmer_CheckForReceipts_confirmed_missing_receipt_scoped_to_key(t
 	mustTxBeInState(t, borm, etx2_9, txmgr.EthTxConfirmed)
 }
 
+func TestEthConfirmer_CheckForReceipts_confirmed_missing_receipt_scoped_to_key(t *testing.T) {
```

Comment:
@samsondav This already exists, the auto conflict resolution by git duplicated this block. The rest of the diffs should be double checked for invalid merges.